### PR TITLE
Np 47183 create auth flow diagram

### PR DESCRIPTION
**Pull request by elkebab**
# Description

Link to Jira issue: https://sikt.atlassian.net/browse/NP-47183

Ny modell for å vise ny autentiseringsflyt, hvor bruker må akseptere brukervilkår før de tar i bruk NVA.

- [Ny flyt](https://github.com/BIBSYSDEV/NVA-Frontend/blob/NP-47183-create-auth-flow-diagram/documentation/authentication.md)
- [Gammel flyt](https://github.com/BIBSYSDEV/NVA-Frontend/blob/develop/documentation/authentication/loginFlow.drawio.png) (`.drawio.png` sliter litt om man bruker dark mode, men man kan se om det blir bedre ved å laste ned bildet)

# Checklist

Ensure that the changes are aligned with the expectations of PO/designer before marking the PR as ready for review.

Also ensure that the following criterias are met:

- [x] The changes are working as expected
- [x] The changes are tested OK for different screen sizes
- [x] The changes are tested OK for a11y
- [x] Interactive elements have data-testids
- [x] I have done a QA of my own changes

_Note: some of these criterias may not always be relevant, and can simply be marked as completed._

### CODE REVIEW

File: documentation/authentication.md
Comment (elkebab) [inline documentation/authentication.md:1]: Se fila i riktig format her:
https://github.com/BIBSYSDEV/NVA-Frontend/blob/425741042919619ca82a075f57d1994ed25187fb/documentation/authentication.md

**Review by brinxmat — DISMISSED** [review]
100% concur that this is the flow that I imagine is correct. I may still be wrong. PO must review.

File: documentation/authentication.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+flowchart TD
+    A[User logs in] --> B{User has previously accepted current terms and conditions?}
+
+    B -- No --> C{User accepts current terms and conditions?}
+    B -- Yes --> D{User exists in Cristin Person Register?}
+
+    C -- No --> E[User is logged out]
+    C -- Yes --> D
+
+    D -- No --> F{User accepts terms for being added to Cristin Person Register?}
```
Comment (elkebab) [inline documentation/authentication.md:18]: JE ville gjerne hatt C og F i samme steg for brukeren. Må ikke nødvendigvis gjenspeiles i denne modellen, men hadde nok vært en fordel.

Comment (elkebab) [reply documentation/authentication.md]: Har slått sammen disse til ett steg nå

**Review by iBuzza — APPROVED** [review]

**Pull request merged**